### ISSUE TITLE
Public View will be only landing page

### ISSUE THREAD

**Issue by EnriqueCorado** [benetech/ServiceNet#1421]
- Currently, we have two landing pages 
https://servicenet.benetech.org/#/
https://servicenet.benetech.org/#/public/BayArea

- Remove/redirect https://servicenet.benetech.org/#/ to https://servicenet.benetech.org/#/public/BayArea
- all CBOs will be logging in through this page on the top right. No need to have two landing pages 

<img width="958" alt="Screenshot 2020-11-23 144941" src="https://user-images.githubusercontent.com/65787629/100024227-3ce8c280-2d9b-11eb-8353-bb12eb6a963e.png">

**Comment by johnhbenetech** [comment]
Two considerations:

1. We need to ensure we have a better understanding of how silos work. There isn't one landing page for servicenet. e.g. Marin will use /Marin and Indiana would use /Indiana so I have a hard time accepting we should just choose one for everyone

2. What happens to the marketing experience? If someone visits servicenet for the first time we are dropping them directly into the interface with no context about what the platform is and who it's for, etc. Things like the marketing video would become obsolete since there's no meaningful way to get there.

I think a better option is to add call to actions to the homepage. In that way we can add obvious callouts to see a demo but still maintain the marketing collateral. We can also choose which silo/geographies to feature like 'Find Bay Area Resources', 'Find St. Louis Resources' or whatever.

**Comment by johnhbenetech** [comment]
Oh also - if you are logged in you will skip the homepage and go right to your silo. So i think we can also rely on sign in to direct people.

I would also encourage you to think about how we can make sign in/registration call to actions more obvious on the home page

**Comment by meenupant** [comment]
Yes, I agree with your points. We will work on the user flows and have one main landing page with all the marketing stuff, public view and CTAs for other user types. The sign in will take the other user types in their silos and if they come to Service Net and already signed in then they will go straight to their silo. I will work on the details and review with you to get the best possible layout and user flows.
